go.mod

```diff
@@ -48,11 +48,11 @@ require (
 	github.com/pingcap/gofail v0.0.0-20181217135706-6a951c1e42c3
 	github.com/pingcap/goleveldb v0.0.0-20171020122428-b9ff6c35079e
 	github.com/pingcap/kvproto v0.0.0-20190110035000-d4fe6b336379
+	github.com/pingcap/log v0.0.0-20190214045112-b37da76f67a7
```

Comment:
https://github.com/pingcap/log/pull/2 is merged, should we update to the latest git hash?

Reply:
It's the latest hash.